packages/playground-ui/src/lib/toast.css

```
@@ -0,0 +1,99 @@
+  --error-border: var(--toast-destructive-border);
+  --warning-bg: var(--toast-warning-bg);
+  --warning-text: var(--notice-warning-fg);
+  --warning-border: var(--toast-warning-border);
+  --info-bg: var(--toast-info-bg);
+  --info-text: var(--notice-info-fg);
+  --info-border: var(--toast-info-border);
+  --border-radius: var(--radius-xl);
+}
+[data-sonner-toaster].mastra-toaster [data-sonner-toast] {
+  box-shadow: var(--toast-shadow);
+}
```

Stylelint (17.9.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)